sacrebleu/sacrebleu.py

```diff
@@ -149,63 +150,63 @@ def main():
         sys.exit(0)
 
     if args.sentence_level and len(args.metrics) > 1:
-        logging.error('Only one metric can be used with Sentence-level reporting.')
+        sacrelogger.error('Only one metric can be used with Sentence-level reporting.')
         sys.exit(1)
 
     if args.citation:
         if not args.test_set:
-            logging.error('I need a test set (-t).')
+            sacrelogger.error('I need a test set (-t).')
             sys.exit(1)
         for test_set in args.test_set.split(','):
             if 'citation' not in DATASETS[test_set]:
-                logging.error('No citation found for %s', test_set)
+                sacrelogger.error('No citation found for %s', test_set)
             else:
                 print(DATASETS[test_set]['citation'])
         sys.exit(0)
 
     if args.num_refs != 1 and (args.test_set is not None or len(args.refs) > 1):
-        logging.error('The --num-refs argument allows you to provide any number of tab-delimited references in a single file.')
-        logging.error('You can only use it with externaly-provided references, however (i.e., not with `-t`),')
-        logging.error('and you cannot then provide multiple reference files.')
+        sacrelogger.error('The --num-refs argument allows you to provide any number of tab-delimited references in a single file.')
+        sacrelogger.error('You can only use it with externaly-provided references, however (i.e., not with `-t`),')
+        sacrelogger.error('and you cannot then provide multiple reference files.')
         sys.exit(1)
 
     if args.test_set is not None:
         for test_set in args.test_set.split(','):
             if test_set not in DATASETS:
-                logging.error('Unknown test set "%s"', test_set)
-                logging.error('Please run with --list to see the available test sets.')
+                sacrelogger.error('Unknown test set "%s"', test_set)
+                sacrelogger.error('Please run with --list to see the available test sets.')
                 sys.exit(1)
 
     if args.test_set is None:
         if len(args.refs) == 0:
-            logging.error('I need either a predefined test set (-t) or a list of references')
-            logging.error(get_available_testsets())
+            sacrelogger.error('I need either a predefined test set (-t) or a list of references')
+            sacrelogger.error(get_a_list_of_testset_names())
```

Comment:
Where is `get_a_list_of_testset_names()` defined? How is this edit related to this PR?

Reply:
Thanks Martin, I merged from a previous version with vimdiff, and it seems I did a wrong copy/paste here. Sorry for that.

Reply:
Now fixed!